shell/src/main/java/alluxio/cli/bundler/command/CollectMetricsCommand.java

```diff
@@ -122,62 +158,46 @@ private void masterMetrics(StringWriter outputBuffer, int i) throws IOException
       // Do not break the loop since the HTTP failure can be due to many reasons
       // Return the error message instead
       LOG.error("Failed to get Alluxio master metrics from URL {}. Exception: ", url, e);
-      metricsResponse =  String.format("Url: %s%nError: %s", url, e.getMessage());
+      metricsResponse =  String.format("{Url: \"%s\",%n\"Error\": %s}", url, e.getMessage());
```

Comment:
im fine with this manual wrapping for now. An alternative is to wrap it with an object like
```
class CollectMetricsResponse {
  String url;
  String error;
  String body;
}
```
Then use jackson ObjectMapper to convert to json. Up to you :)